.github/workflows/dockerImage.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - "develop"
+      - "tsys-release"
```

Comment:
@mebo4b shouldnt this be ONLY tsys-release and not develp?

Reply:
@timomayer this is only the workflow trigger. If you have a look at line 98 you´ll see the condition for executing the Tsys push